docs/wait.md

```diff
@@ -151,10 +156,13 @@ Example (basic usage):
 ``` Lua
 function onLoad()
 	Wait.time(|| print("One"), 1)
+    --> prints "One" after 1 second
```

Comment:
Is there a tabs vs spaces issue here? The alignment is just a bit off, not the end of the world. I'd probably also recommend just `--` rather than `-->`, the former is a standard comment and the later may imply some other convention.